src/gdrive/replace_open.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/bash
+
+# Basic script to replace opn(authorizeUrl, { wait: false }).then(cp => cp.unref()); with process.stdout.write(`Open this URL in your browser: ${authorizeUrl}`);
```

Comment:
Maybe in a follow up, let's add the reasoning you described. future me will thank if this is inside the bash script instead of a github comment :)